src/abstractdataframe/abstractdataframe.jl

```diff
@@ -866,12 +866,16 @@ end
     filter(cols => function, df::AbstractDataFrame)
 
 Return a copy of data frame `df` containing only rows for which `function`
-returns `true`. The function is passed a `DataFrameRow` as its only argument
-if `cols` is not specified.
-If `cols` is passed then elements of column `cols` are passed to the function,
-if `cols` is a single column or as a `NamedTuple` if multiple columns are passed
-(any valid column selector is accepted). Passing `cols` leads to a more efficient
-execution of the operation for large data frames.
+returns `true`.
+
+If `cols` is not specified then he function is passed `DataFrameRow`s as its only argument.
```

Comment:
Mixing plural and singular is weird:
```suggestion
If `cols` is not specified then the function is passed `DataFrameRow`s.
```